python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -226,6 +236,16 @@ def new_resolver_supported?
           python_version >= Python::Version.new("3.7")
         end
 
+        def pip_compile_options_fingerprint(options)
+          options.sub(
+            /--output-file=\S+/, "--output-file=<output_file>"
```

Comment:
Do we need to be concerned with escaped white space in file names?

Reply:
Yeah, this won't catch that. I decided to let that kind of thing be reported separately since it felt like a very edge case, so unlikely to get in the middle too much. We can merge it later if we feel it's best though!